src/utilitarios/sql_catalog.py

```
@@ -1,160 +1,164 @@
+    entries = [
+        SqlCatalogEntry(sql_id=path.relative_to(SQL_ROOT).as_posix(), path=path)
+        for path in _iter_sql_paths(include_archive=include_archive)
+    ]
+    return sorted(entries, key=lambda item: item.sql_id.lower())
+
+
+def get_sql_id(path: Path | str) -> str | None:
+    candidate = Path(path)
+    try:
+        return candidate.resolve().relative_to(SQL_ROOT.resolve()).as_posix()
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)